cmd/bootstrap.go

```diff
@@ -56,6 +58,7 @@ func GetBootstrapCmd() *cobra.Command {
 	cmd.Flags().StringVarP(&flagBootstrapEmail, "email", "e", "admin@gort", "Email for the bootstrapped user")
 	cmd.Flags().StringVarP(&flagBootstrapName, "name", "n", "Gort Administrator", "Full name of the bootstrapped user")
 	cmd.Flags().StringVarP(&flagBootstrapPassword, "password", "p", "", "Password for bootstrapped user (default generated)")
+	allowInsecure = cmd.Flags().Bool("allow_insecure", false, "Permit http URLs to be used")
```

Comment:
Can this use `cmd.Flags.BoolVarP`?

Also, options typically use dash style instead of underscore. The [Gnu style guide](https://www.gnu.org/software/libc/manual/html_node/Argument-Syntax.html) isn't anybody's favorite, but it's a solid standard:

> GNU adds long options to these conventions. Long options consist of ‘--’ followed by a name made of alphanumeric characters and dashes. Option names are typically one to three words long, **with hyphens to separate words**. Users can abbreviate the option names as long as the abbreviations are unique.

Reply:
Ah, that adds an interesting challenge, then. I was trying to keep the name the same between the yaml and the params so the error message always told you to do the correct thing.

Thoughts on how to rectify that?

Reply:
Hmm... would it make sense to have the error message say something like:

> `insecure URL provided, please use https or set the allow insecure flag in the config or client`

?

Reply:
Updated as suggested. Will also submit another CR for the error message change.